Source/Core/BoundingSphere.js

```diff
@@ -1198,6 +1198,35 @@ define([
         return result;
     };
 
+    /**
+     * If the given position is not already within the sphere, projects the given position onto the sphere.
+     * @param {Cartesian3} position The position being projected onto this BoundingSphere.
+     * @param {Cartesian3} [result] The object onto which to store the result.
+     * @returns {Cartesian3} A projected version of the inputted position if it was not originally within the BoundingSphere.
+     */
+    BoundingSphere.prototype.clampToBounds = function(position, result) {
+        //>>includeStart('debug', pragmas.debug);
+        Check.typeOf.object('position', position);
+        //>>includeEnd('debug');
+        result = Cartesian3.clone(position, result);
+
+        // to avoid dividing by zero
```

Comment:
Throughout, remove inline comments like this.  We only use them if something is really non-obvious